pkg/domain/infosync/info.go

```diff
@@ -912,6 +912,21 @@ func (is *InfoSyncer) updateTopologyAliveness(ctx context.Context) error {
 		clientv3.WithLease(is.topologySession.Lease()))
 }
 
+func (is *InfoSyncer) RemoveTopologyInfo() {
```

Comment:
exported method InfoSyncer.RemoveTopologyInfo should have comment or be unexported